public/src/main/java/com/revenuecat/purchases/models/SubscriptionOptions.kt

```diff
@@ -0,0 +1,45 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class SubscriptionOptions(
```

Comment:
Shouldn't `findDefaultOffer` also be part of this class?

Reply:
Yeah I agree with that. 

Aside from that, right now the constructor of this class is also public. Could we make it internal? Like `class SubscriptionOptions internal constructor(...)`. Otherwise we need to add the constructor to the API tests.

Reply:
Ahh, good point! Will make that happen 💪

Reply:
Turns out can't do internal constructor because this is used in the defined in the main module but instantiated in the Google module 😬 

Only thing we could do is make `SubscriptionOptions` and interface and then have a `GoogleSubscriptionOptions` class which... I hate but also think probably makes the most sense 🤔